docs/secondary-network.md

```diff
@@ -313,3 +313,8 @@ spec:
   annotation is added to the Pod for the first time (e.g., when creating the
   Pod), we will configure the secondary network interfaces accordingly, and no
   change is possible after that, until the Pod is deleted.
+* We don't support K8s Nodes with multi-path routes. When the K8s Node interfaces
```

Comment:
I would add a note defining what a multi-path route is. Something like this:

> A multi-path route is a route with multiple possible next hops. When listing the rules (e.g., with `ip route list`), a multi-path route may appear as multiple individual routes (one for each next hop), all with the same cost.

Reply:
Updated the doc and description. Thanks